internal/cmd/inventoryconsumer/cmd.go

```diff
@@ -74,12 +74,6 @@ func handler(ctx context.Context, msg kafka.Message) {
 			logger.Info().Msg("setting up new host for remote host configuration")
 			reqID, _ := util.Kafka.GetHeader(msg, "request_id")
 			logger = logger.With().Str("request_id", reqID).Str("host_id", event.Host.ID).Str("org_id", event.Host.OrgID).Logger()
-			messageID, err := internal.SetupHost(ctx, event.Host)
```

Comment:
After removing these lines, what is difference between `created` and `updated` event_types handling?